### ISSUE TITLE
Document, can't toggle to "+" status

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#18]
I use a lot the "+" status to mark specific items (like all items "+" are in a todo list) :

![image](https://user-images.githubusercontent.com/32078594/103496015-f2983e00-4e3c-11eb-9b9d-d159599aef7e.png)

Unfortunately it doesn't work as before as double clicking toggles from "V" to "X" but not anymore to "+" :-(

**Comment by chiminirc** [comment]
I can confirm this as well.  I'm pretty sure it wasn't present in the first release this year.

**Comment by rgriebl** [comment]
Well, I don't really use the "+", so having to double-click in one direction and quadruple-clicking for the other direction is really odd for me and messed up my lists more than once ;) 
That's why I changed the default behavior in 2021.1.1.
Don't you find that asymmetry between include and exclude confusing?

That being said, you can use the context menu or simply press the keys "X" (tra), "I" (nclude) or "E" (xclude).

**Comment by chiminirc** [comment]
I didn't know the change is intentional. I can live with it as my workflow as I download orders and check them red when they are complete. If only I could filter on that column though.

**Comment by paramecie** [comment]
Thanks for the info!

To clarify, I use it like this (for example). I use CTRL+A and put everything to "+", meaning "to do".
Then I check each lot, and mark "V" if OK, and "X" if not OK.
Sometimes an OK "V" becomes again "+" if I have to re-check.
When I'm done, the whole list shouldn't have any "+" remaining.

Example:
![image](https://user-images.githubusercontent.com/32078594/103542366-1b502000-4e9d-11eb-8021-f8b55e5e8173.png)


Anyway, there's still the menus, and I didn't know the keys x, e, i !

Remarks:

* please change the "x" one, as "x" means "X" more than "+"... Why not "a" as in extrA ?

* please write those shortcuts somewhere, why not in the strings of the submenus, like here:

![image](https://user-images.githubusercontent.com/32078594/103542172-d88e4800-4e9c-11eb-853f-bd99eee92bb3.png)

Include (I)
Exclude (E)
Extra (A)

In short, I'm fine with the change but would appreciate the above remarks to be taken in consideration. Thank you!
